selendroid-test-app/src/main/java/io/selendroid/testapp/WebViewActivity.java

```diff
@@ -42,6 +45,28 @@ public void onCreate(Bundle savedInstanceState) {
     setContentView(io.selendroid.testapp.R.layout.webview);
 
     mainWebView = (WebView) findViewById(io.selendroid.testapp.R.id.mainWebView);
+    WebSettings settings = mainWebView.getSettings();
+    settings.setJavaScriptEnabled(true);
+    settings.setDomStorageEnabled(true); // LocalStorage, SessionStorage
+    
+    // The following is required to enable WebSQL on pre-Kitkat, per
+    // http://pouchdb.com/errors.html#android_pre_kitkat
+    settings.setDatabaseEnabled(true);
+    // On Kitkat and above, the following will be ignored, and it will write 
+    // to the "database" directory anyway.
+    String databasePath = getContext().getApplicationContext().getDir(
+      "database", Context.MODE_PRIVATE).getPath();
```

Comment:
the travis job is failing because of this:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.2:compile (default-compile) on project selendroid-test-app: Compilation failure: Compilation failure:
[ERROR] /home/travis/build/selendroid/selendroid/selendroid-test-app/src/main/java/io/selendroid/testapp/WebViewActivity.java:[58,19] cannot find symbol
[ERROR] symbol:   variable Context